stripe-node/app.js

```diff
@@ -52,31 +50,53 @@ app.post('/charge/angular', (req, res) => {
     source: req.body.token.id,
     shipping: address,
     name: req.body.token.name,
-  })
-  // Add charge to customer
-    .then(customer => stripe.charges.create({
+  }, (err, customers) => {
+    stripe.charges.create({
       amount: req.body.amount,
       currency: req.body.currency,
       description: 'Donation to Keyman',
-      customer: customer.id,
+      customer: customers.id,
       receipt_email: req.body.token.email,
-    }));
-// Successfull api call
-  // .then(res => {
-  //   res.status(200)
-  //     .json({
-  //       status: '200',
-  //       message: res,
-  //     });
-  // })
-  // Handle api failure
-  // .catch(res => {
-  //   res.status(500)
-  //      .json({
-  //        status: '500',
-  //        message: res,
-  //      });
-  // })
+    }, (err, charges) => {
+      // Handle errors
+      console.log(err.type);
+      if (err !== null) {
```

Comment:
This should probably be
```suggestion
      if (typeof err === 'object') {
```

Reply:
Your proposed solution still results in an error when there is no error occurring. I've left it for now since it is working.